subprojects/building-java-libraries/samples/groovy-dsl-files/build.gradle

```diff
@@ -2,19 +2,19 @@
 
 
 plugins {
-    id 'java-library'
+    id 'java-library' // <1>
 }
 
 repositories {
-    jcenter() // <1>
+    jcenter() // <2>
 }
 
 dependencies {
-    api 'org.apache.commons:commons-math3:3.6.1' // <2>
+    api 'org.apache.commons:commons-math3:3.6.1' // <3>
 
-    implementation 'com.google.guava:guava:28.0-jre' // <3>
+    implementation 'com.google.guava:guava:28.0-jre' // <4>
 
-    testImplementation 'junit:junit:4.12' // <4>
+    testImplementation 'junit:junit:4.12' // <5>
```

Comment:
Might be worth upgrading across our documentation

Reply:
It is actually updated already. Gradle 6.5 will use 4.13